crates/config/src/lib.rs

```diff
@@ -1548,8 +1548,14 @@ impl Config {
     /// Returns the configured [VyperSettings] that includes:
     /// - evm version
     pub fn vyper_settings(&self) -> Result<VyperSettings, SolcError> {
+        // Vyper does not yet support Prague, so we normalize it to Cancun.
+        let evm_version = match self.evm_version {
+            EvmVersion::Prague => EvmVersion::Cancun,
+            _ => self.evm_version,
+        };
```

Comment:
should we add `fn normalize_vyper` so that we have this in one single place?

Reply:
Fixed in https://github.com/foundry-rs/foundry/pull/10565/commits/d4a76cbceb2edbaa236e86d19be8e2e90c422d46 

Point of patched test case noted